CONTRIBUTING.md

```diff
@@ -4,6 +4,72 @@ We're so glad you're thinking about contributing to an 18F open source project!
 
 Before contributing, we encourage you to read our CONTRIBUTING policy (you are here), our LICENSE, and our README, all of which should be in this repository. If you have any questions, or want to read more about our underlying policies, you can consult the 18F Open Source Policy GitHub repository at https://github.com/18f/open-source-policy, or just shoot us an email/official government letterhead note to [18f@gsa.gov](mailto:18f@gsa.gov).
 
+## Information
+
+### High-level roadmap
+The cloud.gov deck is currently being refactored to port the codebase from angular to unit-tested react. In doing so, it will take UI/UX cues from the  [community web ui](https://github.com/icclab/cf-webui), and merge that design with the visual [cloudgov-style](https://github.com/18F/cg-) to create a unified cloud.gov experience. This refactor work is accumulated into the `staging-alpha` branch which will eventually be merged with `master` when it's feature complete and bug free. This means all work should be merged into the `staging-alpha` branch, unless working with the deprecated angular codebase.
+
+### Workflow
+Tracking work and progress is currently being done through [Zenhub](https://www.zenhub.io/), which adds a browser extension to add additional agile features to github. The main feature Zenhub adds is a "Boards" page which is an agile bboard detailing the state of work for the deck.
+
+- How epics relate to to stories and how they're tracked with aha.io.
+- How stories should progress through the board.
+- What should a "groomed" story have.
+- What to do with PRs.
+- No need to estimate.
+- Put things in icebox first.
+- How things get prioritized into the backlog.
+
+### Code reviewing
+- Have ready to merge label which anybody can merge. (helps async work)
+- Always have another person review.
+- Linting should happen before review.
+- https://pages.18f.gov/development-guide/code-review/
+- Always merge into `staging-alpha`. Should `staging-alpha` always be deployable?
+- Go back and forth on PRs as only two people.
+- One person at least reviews each PR.
+- What is a good commit message?
+- What to look for in code review?
+- Should code be pulled down and tested for each review?
+- Any time you change a file, you should turn on linting for that file and fix the errors
+- Any new files should be linted.
+- Currently, only main cg-deck repo will work for CI tests, forks can't run it due to privacy issues.
+- How to share credentials required to run things?
+
+#### Current considerations
+- Working off a branch that gets deployed to a non-used site (no feedback on bugs/etc)
```

Comment:
looking into this https://github.com/pivotal-cf/pcfdev so we can test it against a pseudo-live env then creds can be dummy ones.